src/mame/drivers/iphone2g.cpp

```diff
@@ -103,7 +104,7 @@ void iphone2g_spi_device::device_start()
 
 void iphone2g_spi_device::device_reset()
 {
-	cmd = ctrl= status = tx_data = 0;
+	cmd = ctrl = status = tx_data = 0;
```

Comment:
Please try to avoid combined assignments like this. Each assignment should have its own line, like it is in every other driver.